oemof/solph/models.py

```diff
@@ -96,7 +96,8 @@ def __init__(self, es, **kwargs):
 
         # non-negative pyomo variable for all existing flows in energysystem
```

Comment:
I think these lines will break existing models.

Would it be okay to add a flow option like "NonNegative=True" and you can set it to False to make lopf possible?